utensor_cgen/backend/operators.py

```diff
@@ -46,6 +55,9 @@ def snippet(self):
 
 
 @OperatorFactory.register
+@OpEqualityDelegate.is_associative(
+  permutations=((0, 1), (1, 0))
```

Comment:
Does this support groupings? For example;
`q_add(a, a_min, a_max, b, b_min, b_max)` == `q_add( b, b_min, b_max, a, a_min, a_max)`

Reply:
I don't get it.
The later can be configured as `(3, 4, 5, 0, 1, 2)`.
Well, a tuple is kinda a formal math notation for permutation.

Reply:
All you have to do is to list all possible permutations in this decorator.

Reply:
Ok. As long as this supports multiple permutations. It shouldn’t be a problem since the numbers of operants are usually small.